### PR TITLE
docs: link to OARC oss-vulns list; add security policy to dnsdist; include GPL license; some minor refactoring

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Review by rgacogne — APPROVED** [review]

**Comment by Habbie** [comment]
`checking consistency... /home/peter/projects/powerdns/pdns/pdns/dnsdistdist/docs/security-advisories/security-policy.rst: WARNING: document isn't included in any toctree` I'll fix that

**Review by rgacogne — APPROVED** [review]
:+1:

File: docs/common/security-policy.rst [hunk 1]
```diff
@@ -7,9 +7,11 @@ In case you want to encrypt your report using PGP, please use: https://www.power
 Please do not mail security issues to public lists, nor file a ticket, unless we do not get back to you in a timely manner.
 We fully credit reporters of security issues, and respond quickly, but please allow us a reasonable timeframe to coordinate a response.
 
-We remind PowerDNS users that under the terms of the GNU General Public License, PowerDNS comes with ABSOLUTELY NO WARRANTY.
+We remind PowerDNS and dnsdist users that under the terms of the GNU General Public License, PowerDNS and dnsdist come with ABSOLUTELY NO WARRANTY.
 This license is included in this documentation.
```
Comment (pieterlexis) [inline docs/common/security-policy.rst:11]: slightly offtopic, but this sentence is a lie :)

Comment (Habbie) [reply docs/common/security-policy.rst:11]: oh, the license is missing? onward!

**Pull request merged**